### ISSUE TITLE
It can be updated? 

### ISSUE THREAD

**Issue by openclues** [thlorenz/batufo#11]
Hello
It is a really great project, but I can not run it at all on the new version of flutter.

Can you please update it?

**Comment by patzu** [comment]
Yes it is not compiling lots of errors. Anyway I have a question, How to shoot toward any direction I can't figure out how to calculate and find how much I should add to bullets position after changing angle of rifle?
